packages/mermaid/src/rendering-util/rendering-elements/edges.js

```diff
@@ -592,11 +564,6 @@ export const insertEdge = function (elem, edge, clusterDb, diagramType, startNod
   if (edge.look === 'handdrawn') {
     const rc = rough.svg(elem);
     Object.assign([], lineData);
-    // const svgPathNode = rc.path(lineFunction(ld.splice(0, ld.length-1)), {
```

Comment:
Leave the lineFunction for now, we can remove it in the  big PR. 
Still working out some issue around it.